apps/widhrt2/widget.js

```diff
@@ -0,0 +1,49 @@
+(function(){
+  if (!Bangle.isHRMOn) return; // old firmware
+  var hp = Bangle.setHRMPower;
+  Bangle.setHRMPower = () => {
```

Comment:
I think we can get away with this on Espruino JS, but `() => ...` functions don't have `arguments`, so we should perhaps call this a `function` to avoid breaking if Espruino changes to match the spec more

```suggestion
  Bangle.setHRMPower = function() {
```

Reply:
Intriguing,  will functions declared like so break:

`(arg1)=>{...}`

?

Reply:
So with explicit arguments, do you mean? For example:

```javascript
  Bangle.setHRMPower = (isOn, appID) => {
    hp.call(Bangle, isOn, appID);
    WIDGETS.widhrt.draw();
  };
```

Yes, that would be fine in this case - I suppose to make things behave as transparent as possible we should forward on the `this` value rather than hardcoding it as `Bangle` here, but you could say that's getting too academic